docs/explanation/cryptography.rst

```diff
@@ -0,0 +1,20 @@
+.. _explanation_cryptographic-technology:
+
+Cryptographic technology in Craft Providers
+===========================================
+
+Craft Providers uses cryptographic processes to communicate with local
+processes. It does not directly implement its own cryptography, but it does
```

Comment:
> processes to communicate with local processes

Combining processes with locality sounds odd to me. Is there a more standard way we could describe this?

Maybe I'm hung up on "processes". Do you mean in the program sense? If not, I suggest using a synonym instead.

Reply:
I like your idea. What about:

> Craft Providers uses cryptographic technologies to communicate with processes running on the host system.